tests/middleware/test_base.py

```diff
@@ -50,13 +42,24 @@ async def dispatch(self):
         pass
 
 
-@app.websocket_route("/ws")
 async def websocket_endpoint(session):
     await session.accept()
     await session.send_text("Hello, world!")
     await session.close()
 
 
+app = Starlette(
+    routes=[
+        Route("/", endpoint=homepage),
+        Route("/exc", endpoint=exc),
+        Route("/exc-stream", endpoint=exc_stream),
+        Route("/no-response", endpoint=NoResponse),
+        WebSocketRoute("/ws", endpoint=websocket_endpoint),
+    ]
+)
+app.add_middleware(CustomMiddleware)
```

Comment:
Could we have the middleware included with `Starlette(middleware=...)`?

Reply:
Done 👍